blake3.tex

```diff
@@ -428,6 +428,10 @@ \subsection{Modes}\label{sec:modes}
 the first stage, and the \flag{DERIVE_KEY_MATERIAL} flag set for every
 compression.
 
+Developers adapting BLAKE3 to ASN.1-based message formats should use
+the Algorithm Identifier blake3 with OID identifier
+1.3.6.1.4.1.1722.12.3.8 for all modes and 256-bit default output size.
+
```

Comment:
@oconnor663 based on discussion here, and the https://github.com/BLAKE3-team/BLAKE3/issues/68 I've updated this patch.

Moved the notice to the Modes section. Specifying that the same OID applies to all modes, for the default output size of 256-bit. Which matches the description of the previous sections which define the default output size.